Modules/pyexpat.c

```diff
@@ -1878,6 +1878,13 @@ add_features(PyObject *mod)
 }
 #endif
 
+static void
+pyexpat_capsule_destructor(PyObject *capsule)
+{
+    PyMem_Free(PyCapsule_GetPointer(capsule, PyExpat_CAPSULE_NAME));
```

Comment:
If `PyCapsule_GetPointer` fails it will set an exception. We should check that, and `PyErr_WriteUnraisable` on error.

Reply:
Good catch added.